### PR TITLE
Do not include area of air wall surfaces in floor area

### PR DESCRIPTION
**Pull request by macumber**
@DavidGoldwasser @asparke2 what do you think about this?

### CODE REVIEW

**Comment by macumber** [comment]
There are probably more things I'd need to do here (like make sure to remove loads on E+ translation) but what do you think about the general idea of not counting air walls in floor area?  I think David may have a ticket or two about this

**Comment by DavidGoldwasser** [comment]
@macumber I like this idea. Issue #561 about stacked spaces relate to this. It is important for per area loads to be accounted for as well.

**Comment by macumber** [comment]
@asparke2 @DavidGoldwasser this is ready for testing, should I merge this in for 1.12.3 for testing?  Should I ask @evanweaver to roll a separate build?

File: openstudiocore/src/model/ThermalZone.cpp [hunk 1]
```diff
@@ -1246,11 +1274,16 @@ namespace detail {
...
     if (needToSetFloorArea){
+      
+      // do not allow per area loads in the space type since we are overriding floor area
+      spaceType.reset();
+
+      // don't override if user provided zone floor area
       if (isEmpty(OS_ThermalZoneFields::FloorArea)){
-        // DLM: do we want to do this?
-        //this->setDouble(OS_ThermalZoneFields::FloorArea, totalFloorArea);
+        this->setDouble(OS_ThermalZoneFields::FloorArea, totalFloorArea);
```
Comment (macumber) [inline openstudiocore/src/model/ThermalZone.cpp]: This will write to the zone's floor area field in E+

**Comment by evanweaver** [comment]
Test builds under way.

**Pull request merged**